common/djangoapps/enrollment/views.py

```diff
@@ -616,13 +616,14 @@ def post(self, request):
                 org = course_id.org
                 update_email_opt_in(request.user, org, email_opt_in)
 
+            log.info('The user %s has already been enrolled in course run %s.', username, course_id)
```

Comment:
@ahsan-ul-haq I realize this is a nitpick, but can you please keep this consistent with the other messages you're adding? I think what you had [here](https://github.com/edx/edx-platform/pull/13663/files#diff-a4a6f81c03f035ebb4441f342e27c62eR623) was perfect.